<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,30">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,30,1,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 160, 174);</string>
            </property>
            <property name="text">
             <string>设备列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DeviceListWidget" name="listWidget"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(80, 160, 174);</string>
            </property>
            <property name="text">
             <string>云台控制</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PtzWidget" name="ptzwidget" native="true">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(85, 87, 83);</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="20,6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="VideoWidget" name="openGLWidget"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,30">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(80, 160, 174);</string>
              </property>
              <property name="text">
               <string>报警信息</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>PtzWidget</class>
   <extends>QWidget</extends>
   <header>ptzwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DeviceListWidget</class>
   <extends>QListWidget</extends>
   <header>devicelistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
